dist/_patterns/00-atoms/05-forms/00-text-fields.twig

```diff
@@ -1,4 +1,3 @@
-{% import "@macros/forms.twig" as forms %}
```

Comment:
Explicit import not necessary per https://github.com/pattern-lab/patternengine-php-twig#macros.

`forms.twig` renamed to `forms.macro.twig` per the macro extension defined in the `config.yml`.